apps/regression/model/linear_model.cpp

```diff
@@ -28,7 +27,7 @@ double LinearModel::levelSet(double * modelCoefficients, double xMin, double ste
   if (a == 0) {
```

Comment:
As you are modifying this file anyways, could you please change this to `a == 0.0` ? Thanks!